README.md

```diff
@@ -18,6 +18,8 @@ Download the official installer for your operating system:
 You can install this alongside your existing GitHub Desktop for Mac or GitHub
 Desktop for Windows application.
 
+For more information on Github Desktop please visit this [article](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop)
```

Comment:
Since we already have a "More resources" section in the readme I think it would make sense to move this paragraph underneath that and to tweak it to match the other link already in that section

```suggestion
See our [getting started documentation](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop) for more information on how to set up, authenticate, and configure GitHub Desktop.
```

Could you make that change and then move it to the more resources section?